### PR TITLE
update tool chromeister to 1.3.c with new requirements

### PR DESCRIPTION
**Pull request by estebanpw**
This pull request is to update CHROMEISTER to version 1.3.c **NOTE: I edited this to 1.3.c because of further updates**
This new version requires python and a few packages, can please someone check if I wrote the build.sh and the meta.yaml correctly?

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/chromeister/meta.yaml [hunk 1]
```diff
@@ -16,14 +16,38 @@ requirements:
...
+    - opencv
+    - Pillow
+    - pyparsing
+    - python-dateutil
+    - six
 
 test:
   commands:
     - CHROMEISTER --help 2>&1 > /dev/null
+    - python detect_events.py > /dev/null
```
Comment (bernt-matthias) [inline recipes/chromeister/meta.yaml:50]: You should make it a "real executable":

- add `#!/usr/bin/env python` as first line in the script
- make it executable `chmod ugo+x detect_events.py`
- call it as `detect_events.py`

As a side note the: python script has some unused matplotlib imports which can be removed... are all the new requirements needed here?

Comment (bernt-matthias) [reply recipes/chromeister/meta.yaml:50]: I guess most of the requirements are only needed in the run section, if at all.

Comment (estebanpw) [reply recipes/chromeister/meta.yaml]: Thanks a lot for your help @bernt-matthias !

I just made it executable and removed matplotlib, thanks for letting me know

Comment (estebanpw) [reply recipes/chromeister/meta.yaml]: > I guess most of the requirements are only needed in the run section, if at all.

OK, let me see if the tests work now and then I remove them

**Comment by estebanpw** [comment]
Currently its failing with 
```
13:26:47 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/chromeister_1607001445731/test_tmp/run_test.py", line 2, in <module>
13:26:47 BIOCONDA INFO (OUT)     import cv2
13:26:47 BIOCONDA INFO (OUT) ImportError: libGL.so.1: cannot open shared object file: No such file or directory
13:26:47 BIOCONDA INFO (OUT) import: 'cv2'
```

Not sure how to fix it. I have tried several things (see past commits)
Any idea @bernt-matthias ?

**Comment by bernt-matthias** [comment]
Does this info help: https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl

**Comment by dpryan79** [comment]
I'm assuming that this builds some python packages that are installed in the normal way. If that's not the case and it just (A) needs python (3?) for installation and to run some scripts in $PREFIX/bin then let me know and we can cut the build time down a lot.

**Comment by estebanpw** [comment]
> 
> 
> I'm assuming that this builds some python packages that are installed in the normal way. If that's not the case and it just (A) needs python (3?) for installation and to run some scripts in $PREFIX/bin then let me know and we can cut the build time down a lot.
 
For installation I think it only really needs the C compiler (to compile CHROMEISTER) (see the `build.sh` file). 
After that there is a script called `detect_events.py` in the `bin` folder which requires python3 and a few packages (`opencv` among them, which is the one that wants `libGL`) that I normally install with pip. 

I put all requirements in both build and run because I wasnt sure whether packages should be gathered in the build step. Let me know if this answers your question

**Comment by estebanpw** [comment]
Btw, in one of my attempts to make it work I used `opencv <4` but I would rather change this to just `opencv` if it doesnt help. Not sure if that will interfere with the rest of things.

**Comment by dpryan79** [comment]
let's let the current one finish and if that works we can remove the version restriction on opencv

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
Looks good to me now. If you're happy with thing, post a message that starts with `@bioconda-bot please merge` and this will get merged in.

**Comment by estebanpw** [comment]
> 
> 
> Looks good to me now. If you're happy with thing, post a message that starts with `@bioconda-bot please merge` and this will get merged in.

looks great to me! Thank you so much for your help

**Comment by estebanpw** [comment]
@bioconda-bot please merge

**Pull request merged**